src/pages/mainPage/sectionAbout/SectionAbout.tsx

```diff
@@ -1,9 +1,9 @@
 export const SectionAbout = function () {
   return (
     <section className="section2">
-        <div className="about-img">
-            <img src="/img/about/about-pets.png" alt=""/>
-        </div>
+      <div className="about-img">
+        <img src="/img/about/about-pets.png" alt="" />
```

Comment:
always add alts
they are used for auto to read the site content outloud for blind people